### PR TITLE
 CONSOLE-3772: Add CI jobs for crontab-plugin repository

### PR DESCRIPTION
**Pull request by jhadvig**
/assign @spadgett 

@rhamilto FYI

### CODE REVIEW

File: ci-operator/jobs/openshift/ci-tools/openshift-ci-tools-master-postsubmits.yaml [hunk 1]
```diff
@@ -1,31 +1,5 @@
 postsubmits:
   openshift/ci-tools:
-  - agent: kubernetes
```
Comment (spadgett) [inline ci-operator/jobs/openshift/ci-tools/openshift-ci-tools-master-postsubmits.yaml:3]: Do we know why there were changes in this file? I assume it's a linter automatically reordering this?

**Review by spadgett — COMMENTED** [review]
No major concerns, just one question on why an unrelated file was touched. We'll need test platform team approval on the PR.

File: ci-operator/jobs/openshift/ci-tools/openshift-ci-tools-master-postsubmits.yaml (hunk 1 above)
Comment (jhadvig) [reply ci-operator/jobs/openshift/ci-tools/openshift-ci-tools-master-postsubmits.yaml:3]: @spadgett yes this was automatic change.

**Comment by jhadvig** [comment]
@spadgett I've rebased the PR. Should be good to go :)

File: core-services/image-mirroring/openshift/mapping_origin_4_14 [hunk 2]
```diff
@@ -104,6 +104,7 @@ registry.ci.openshift.org/origin/4.14:container-networking-plugins quay.io/opens
 registry.ci.openshift.org/origin/4.14:coredns quay.io/openshift/origin-coredns:4.14 quay.io/openshift/origin-coredns:4.14.0
 registry.ci.openshift.org/origin/4.14:crd-schema-checker-tools quay.io/openshift/origin-crd-schema-checker-tools:4.14 quay.io/openshift/origin-crd-schema-checker-tools:4.14.0
 registry.ci.openshift.org/origin/4.14:crd-schema-gen quay.io/openshift/origin-crd-schema-gen:4.14 quay.io/openshift/origin-crd-schema-gen:4.14.0
+registry.ci.openshift.org/origin/4.14:crontab-plugin quay.io/openshift/origin-crontab-plugin:4.14 quay.io/openshift/origin-crontab-plugin:4.14.0
```
Comment (bparees) [inline core-services/image-mirroring/openshift/mapping_origin_4_14:107]: plugin for what?  please include "console" in the name of this image.

Comment (bparees) [reply core-services/image-mirroring/openshift/mapping_origin_4_14:107]: see `quay.io/openshift/origin-nmstate-console-plugin` for example.

**Comment by jhadvig** [comment]
@bparees I've updated the repository name to `console-crontab-plugin` and also CI config accordingly. PTAL :)

**Review by spadgett — APPROVED** [review]
/lgtm

**Pull request merged**